spec/lib/msf/ui/text/dispatcher_shell_spec.rb

```diff
@@ -1,32 +1,264 @@
 require 'spec_helper'
+require 'readline'
 
 RSpec.describe Rex::Ui::Text::DispatcherShell do
-  let(:prompt) { "%undmsf6%clr" }
-  let(:prompt_char) { "%clr>" }
+  let(:prompt) { '%undmsf6%clr' }
+  let(:prompt_char) { '%clr>' }
   let(:subject) do
     dummy_class = Class.new
     dummy_class.include described_class
     dummy_class.new(prompt, prompt_char)
   end
 
+  let(:mock_dispatcher) do
+    dispatcher = double :mock_dispatcher
+    allow(dispatcher).to receive(:tab_complete_helper) do |_current_word, _preceding_words|
+      ['username=']
```

Comment:
shouldn't `password=` be in here aswell since it's an option that _should_ be available for completion in the tests below?